### PR TITLE
Changed app message

### PR DESCRIPTION
**Pull request by frank-the-tank-1**

### CODE REVIEW

File: files/deploy_app.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ cat << EOM > /var/www/html/index.html
   <!-- BEGIN -->
   <center><img src="http://${PLACEHOLDER}/${WIDTH}/${HEIGHT}"></img></center>
   <center><h2>Meow World!</h2></center>
-  Welcome to ${PREFIX}'s app. Replace this text with your own.
+  Whoa! What a cool app! Thanks, ${PREFIX} for creating such an amazing app!
```
Comment (cstella84) [inline files/deploy_app.sh:14]: Wow, what a great change!

File: files/deploy_app.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ cat << EOM > /var/www/html/index.html
   <!-- BEGIN -->
   <center><img src="http://${PLACEHOLDER}/${WIDTH}/${HEIGHT}"></img></center>
```
Comment (cstella84) [inline files/deploy_app.sh:12]: No changes here

**Review by cstella84 — APPROVED** [review]

**Review by cstella84 — COMMENTED** [review]
This is my review

**Comment by frank-the-tank-1** [comment]
Adding a comment

**Pull request merged**